### PR TITLE
Add an indicator when a new page is being loaded

### PR DESCRIPTION
**Pull request by jamiemccue**
Added a page loader indicator when a new page is being loaded.

### CODE REVIEW

File: static/loading-bar.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    this.meter = $('<span>', {style: 'width:0%;'});
+    this.elem = $('<div>', {style: 'display:none', class: 'meter animate '+c}).append(
+      this.meter.append($('<span>'))
+  );
+  $('body').prepend(this.elem);
+    return this;
+}
+
+LoadingBar.prototype.start = function(){
+    this.meter.css('width', '10%');
```
Comment (chasenlehara) [inline static/loading-bar.js]: I like that it starts off with _some_ width albeit I have seen it jump back on long pages (like _Technical Highlights_). Maybe we could make sure it doesn’t dip below 10 once it’s been started?

**Comment by jamiemccue** [comment]
@chasenlehara I revised the loader to start at 1%

**Comment by chasenlehara** [comment]
@jamiemccue I renamed this so it’s easier to understand what this PR is for. I’ll take a look at it soon. :)

**Comment by chasenlehara** [comment]
@imaustink Could you look this over one more time before merging please?

**Pull request merged**